spec/controllers/comments_controller_spec.rb

```diff
@@ -446,6 +446,32 @@
         end
       end
     end
+
+    context "guest comments are turned on in admin settings" do
+      let(:work) { create(:work) }
```

Comment:
Let's set the `AdminSetting` here as well -- even though `guest_comments_off: false` is the default, it's probably safer/a good idea to make it explicit.